api/proto/config/appconfig.proto

```diff
@@ -63,5 +82,8 @@ message AppInstanceConfig {
   CipherBlock cipherData = 13;
 
   // UUID of volumes to be attached to the app-instance.
+  // The order here is critical because they are presented to the VM or
+  // container in the order they are listed, e.g., the first VM image
+  // will be the root disk.
   repeated string volumeRefs = 14;
```

Comment:
Do we need a purge if these change? Or just restart needed? Please add a comment here

Reply:
It says "drives/volumeRefs" in the comment at the top. I don't think we need to repeat that for every field. We have a sufficiently hard time to keep the few comments we have up to date with the behavior and duplication won't make that easier.